Models/Database/Session.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace CoviIDApiCore.Models.Database
+{
+    public class Session : BaseModel<Guid>
+    {
+        public virtual Wallet Wallet { get; set; }
+        public DateTime ExpireAt { get; set; }
```

Comment:
Maybe add CreatedAt as well.

Reply:
There is a createdAt in the wallet Database, but ill add it